src/JustSaying/JustSayingBus.cs

```diff
@@ -55,6 +57,10 @@ public sealed class JustSayingBus : IMessagingBus, IMessagePublisher, IDisposabl
         Config = config;
         SerializationRegister = serializationRegister;
         MiddlewareMap = new MiddlewareMap();
+        _decompressionRegistry = new MessageDecompressionRegistry(new List<IMessageBodyDecompressor>
+        {
+            new GzipMessageBodyDecompressor()
+        });
```

Comment:
Could probably condense this to this?

```suggestion
        _decompressionRegistry = new MessageDecompressionRegistry([new GzipMessageBodyDecompressor()]);
```